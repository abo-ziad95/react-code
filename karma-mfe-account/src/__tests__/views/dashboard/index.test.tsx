import Karma from "@hatech/karma-core";
import React from "react";
import renderer from "react-test-renderer";
import DashboardView from "../../../views/dashboard";

test("ApplyView to make sure it properly renders", () => {
  const component = renderer.create(
    <Karma layout="account">
      <DashboardView />
    </Karma>
  );
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
