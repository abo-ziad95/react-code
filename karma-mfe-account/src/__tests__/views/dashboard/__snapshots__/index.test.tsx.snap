// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ApplyView to make sure it properly renders 1`] = `
<div
  className="makeStyles-root-2"
>
  <header
    className="MuiPaper-root MuiPaper-elevation4 MuiAppBar-root MuiAppBar-positionFixed makeStyles-appBar-4 MuiAppBar-colorPrimary mui-fixed"
  >
    <div
      className="MuiToolbar-root MuiToolbar-regular MuiToolbar-gutters"
    >
      <div
        className="makeStyles-logo-5"
      >
        <a
          href="http://localhost"
          style={
            Object {
              "textDecoration": "none",
            }
          }
        >
          <img
            alt="Karma Careers"
            height="24px"
            src="https://karma-careers-uploads.s3-us-west-2.amazonaws.com/logo.png"
          />
        </a>
      </div>
      <div
        className="makeStyles-menu-47"
      >
        <button
          className="MuiButtonBase-root MuiButton-root MuiButton-text MuiButton-colorInherit"
          disabled={false}
          onBlur={[Function]}
          onClick={[Function]}
          onDragLeave={[Function]}
          onFocus={[Function]}
          onKeyDown={[Function]}
          onKeyUp={[Function]}
          onMouseDown={[Function]}
          onMouseLeave={[Function]}
          onMouseUp={[Function]}
          onTouchEnd={[Function]}
          onTouchMove={[Function]}
          onTouchStart={[Function]}
          tabIndex={0}
          type="button"
        >
          <span
            className="MuiButton-label"
          >
            Post Job
          </span>
        </button>
        <button
          className="MuiButtonBase-root MuiButton-root MuiButton-outlined MuiButton-colorInherit"
          disabled={false}
          onBlur={[Function]}
          onClick={[Function]}
          onDragLeave={[Function]}
          onFocus={[Function]}
          onKeyDown={[Function]}
          onKeyUp={[Function]}
          onMouseDown={[Function]}
          onMouseLeave={[Function]}
          onMouseUp={[Function]}
          onTouchEnd={[Function]}
          onTouchMove={[Function]}
          onTouchStart={[Function]}
          tabIndex={0}
          type="button"
        >
          <span
            className="MuiButton-label"
          >
            Login
          </span>
        </button>
      </div>
    </div>
  </header>
  <div
    className="MuiDrawer-root MuiDrawer-docked makeStyles-drawer-83"
  >
    <div
      className="MuiPaper-root MuiPaper-elevation0 MuiDrawer-paper makeStyles-drawerPaper-85 MuiDrawer-paperAnchorLeft MuiDrawer-paperAnchorDockedLeft"
    >
      <ul
        className="MuiList-root MuiList-padding"
      >
        <li
          className="MuiListItem-root makeStyles-toolbar-100 MuiListItem-gutters"
          disabled={false}
        />
        <li
          className="MuiListSubheader-root MuiListSubheader-sticky MuiListSubheader-gutters"
        >
          Account Menu
        </li>
        <div
          aria-disabled={false}
          className="MuiButtonBase-root MuiListItem-root MuiListItem-gutters MuiListItem-divider MuiListItem-button"
          onBlur={[Function]}
          onClick={[Function]}
          onDragLeave={[Function]}
          onFocus={[Function]}
          onKeyDown={[Function]}
          onKeyUp={[Function]}
          onMouseDown={[Function]}
          onMouseLeave={[Function]}
          onMouseUp={[Function]}
          onTouchEnd={[Function]}
          onTouchMove={[Function]}
          onTouchStart={[Function]}
          role="button"
          tabIndex={0}
        >
          <div
            className="MuiListItemIcon-root"
          >
            <svg
              aria-hidden="true"
              className="MuiSvgIcon-root"
              focusable="false"
              role="presentation"
              viewBox="0 0 24 24"
            >
              <path
                d="M20 6h-8l-2-2H4c-1.1 0-1.99.9-1.99 2L2 18c0 1.1.9 2 2 2h16c1.1 0 2-.9 2-2V8c0-1.1-.9-2-2-2zm-5 3c1.1 0 2 .9 2 2s-.9 2-2 2-2-.9-2-2 .9-2 2-2zm4 8h-8v-1c0-1.33 2.67-2 4-2s4 .67 4 2v1z"
              />
            </svg>
          </div>
          <div
            className="MuiListItemText-root"
          >
            <span
              className="MuiTypography-root MuiListItemText-primary MuiTypography-body1"
            >
              Dashboard
            </span>
          </div>
        </div>
        <div
          aria-disabled={false}
          className="MuiButtonBase-root MuiListItem-root MuiListItem-gutters MuiListItem-divider MuiListItem-button"
          onBlur={[Function]}
          onClick={[Function]}
          onDragLeave={[Function]}
          onFocus={[Function]}
          onKeyDown={[Function]}
          onKeyUp={[Function]}
          onMouseDown={[Function]}
          onMouseLeave={[Function]}
          onMouseUp={[Function]}
          onTouchEnd={[Function]}
          onTouchMove={[Function]}
          onTouchStart={[Function]}
          role="button"
          tabIndex={0}
        >
          <div
            className="MuiListItemIcon-root"
          >
            <svg
              aria-hidden="true"
              className="MuiSvgIcon-root"
              focusable="false"
              role="presentation"
              viewBox="0 0 24 24"
            >
              <path
                d="M18 8c0-3.31-2.69-6-6-6S6 4.69 6 8c0 3.49 3.62 8.19 5.23 10.12.4.48 1.13.48 1.53 0C14.38 16.19 18 11.49 18 8zm-8 0c0-1.1.9-2 2-2s2 .9 2 2-.89 2-2 2c-1.1 0-2-.9-2-2zM5 21c0 .55.45 1 1 1h12c.55 0 1-.45 1-1s-.45-1-1-1H6c-.55 0-1 .45-1 1z"
              />
            </svg>
          </div>
          <div
            className="MuiListItemText-root"
          >
            <span
              className="MuiTypography-root MuiListItemText-primary MuiTypography-body1"
            >
              Explore
            </span>
          </div>
        </div>
        <div
          aria-disabled={false}
          className="MuiButtonBase-root MuiListItem-root MuiListItem-gutters MuiListItem-divider MuiListItem-button"
          onBlur={[Function]}
          onClick={[Function]}
          onDragLeave={[Function]}
          onFocus={[Function]}
          onKeyDown={[Function]}
          onKeyUp={[Function]}
          onMouseDown={[Function]}
          onMouseLeave={[Function]}
          onMouseUp={[Function]}
          onTouchEnd={[Function]}
          onTouchMove={[Function]}
          onTouchStart={[Function]}
          role="button"
          tabIndex={0}
        >
          <div
            className="MuiListItemIcon-root"
          >
            <svg
              aria-hidden="true"
              className="MuiSvgIcon-root"
              focusable="false"
              role="presentation"
              viewBox="0 0 24 24"
            >
              <path
                d="M12 12c2.21 0 4-1.79 4-4s-1.79-4-4-4-4 1.79-4 4 1.79 4 4 4zm0 2c-2.67 0-8 1.34-8 4v2h16v-2c0-2.66-5.33-4-8-4z"
              />
            </svg>
          </div>
          <div
            className="MuiListItemText-root"
          >
            <span
              className="MuiTypography-root MuiListItemText-primary MuiTypography-body1"
            >
              Profile
            </span>
          </div>
        </div>
        <div
          aria-disabled={false}
          className="MuiButtonBase-root MuiListItem-root MuiListItem-gutters MuiListItem-divider MuiListItem-button"
          onBlur={[Function]}
          onClick={[Function]}
          onDragLeave={[Function]}
          onFocus={[Function]}
          onKeyDown={[Function]}
          onKeyUp={[Function]}
          onMouseDown={[Function]}
          onMouseLeave={[Function]}
          onMouseUp={[Function]}
          onTouchEnd={[Function]}
          onTouchMove={[Function]}
          onTouchStart={[Function]}
          role="button"
          tabIndex={0}
        >
          <div
            className="MuiListItemIcon-root"
          >
            <svg
              aria-hidden="true"
              className="MuiSvgIcon-root"
              focusable="false"
              role="presentation"
              viewBox="0 0 24 24"
            >
              <path
                d="M0 0h20v20H0V0z"
                fill="none"
                transform="scale(1.2, 1.2)"
              />
              <path
                d="M15.95 10.78c.03-.25.05-.51.05-.78s-.02-.53-.06-.78l1.69-1.32c.15-.12.19-.34.1-.51l-1.6-2.77c-.1-.18-.31-.24-.49-.18l-1.99.8c-.42-.32-.86-.58-1.35-.78L12 2.34c-.03-.2-.2-.34-.4-.34H8.4c-.2 0-.36.14-.39.34l-.3 2.12c-.49.2-.94.47-1.35.78l-1.99-.8c-.18-.07-.39 0-.49.18l-1.6 2.77c-.1.18-.06.39.1.51l1.69 1.32c-.04.25-.07.52-.07.78s.02.53.06.78L2.37 12.1c-.15.12-.19.34-.1.51l1.6 2.77c.1.18.31.24.49.18l1.99-.8c.42.32.86.58 1.35.78l.3 2.12c.04.2.2.34.4.34h3.2c.2 0 .37-.14.39-.34l.3-2.12c.49-.2.94-.47 1.35-.78l1.99.8c.18.07.39 0 .49-.18l1.6-2.77c.1-.18.06-.39-.1-.51l-1.67-1.32zM10 13c-1.65 0-3-1.35-3-3s1.35-3 3-3 3 1.35 3 3-1.35 3-3 3z"
                transform="scale(1.2, 1.2)"
              />
            </svg>
          </div>
          <div
            className="MuiListItemText-root"
          >
            <span
              className="MuiTypography-root MuiListItemText-primary MuiTypography-body1"
            >
              Settings
            </span>
          </div>
        </div>
        <div
          aria-disabled={false}
          className="MuiButtonBase-root MuiListItem-root MuiListItem-gutters MuiListItem-divider MuiListItem-button"
          onBlur={[Function]}
          onClick={[Function]}
          onDragLeave={[Function]}
          onFocus={[Function]}
          onKeyDown={[Function]}
          onKeyUp={[Function]}
          onMouseDown={[Function]}
          onMouseLeave={[Function]}
          onMouseUp={[Function]}
          onTouchEnd={[Function]}
          onTouchMove={[Function]}
          onTouchStart={[Function]}
          role="button"
          tabIndex={0}
        >
          <div
            className="MuiListItemIcon-root"
          >
            <svg
              aria-hidden="true"
              className="MuiSvgIcon-root"
              focusable="false"
              role="presentation"
              viewBox="0 0 24 24"
            >
              <path
                d="M18 8h-1V6c0-2.76-2.24-5-5-5S7 3.24 7 6v2H6c-1.1 0-2 .9-2 2v10c0 1.1.9 2 2 2h12c1.1 0 2-.9 2-2V10c0-1.1-.9-2-2-2zm-6 9c-1.1 0-2-.9-2-2s.9-2 2-2 2 .9 2 2-.9 2-2 2zm3.1-9H8.9V6c0-1.71 1.39-3.1 3.1-3.1 1.71 0 3.1 1.39 3.1 3.1v2z"
              />
            </svg>
          </div>
          <div
            className="MuiListItemText-root"
          >
            <span
              className="MuiTypography-root MuiListItemText-primary MuiTypography-body1"
            >
              Logout
            </span>
          </div>
        </div>
      </ul>
    </div>
  </div>
  <main
    className="makeStyles-content-1"
  >
    <div
      className="makeStyles-toolbar-3"
    />
    <div
      className="wrapper"
    >
      <div
        className="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-3"
      >
        <div
          className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-12 MuiGrid-grid-md-12 MuiGrid-grid-lg-12 MuiGrid-grid-xl-12"
        >
          <div
            className="makeStyles-breadcrumbs-169"
          >
            <nav
              className="MuiTypography-root MuiBreadcrumbs-root MuiTypography-body1 MuiTypography-colorTextSecondary"
            >
              <ol
                className="MuiBreadcrumbs-ol"
              >
                <li
                  className="MuiBreadcrumbs-li"
                >
                  <a
                    className="MuiTypography-root MuiLink-root MuiLink-underlineHover MuiTypography-colorInherit"
                    onBlur={[Function]}
                    onClick={[Function]}
                    onFocus={[Function]}
                  >
                    Dashboard
                  </a>
                </li>
              </ol>
            </nav>
          </div>
        </div>
        <div
          className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-12 MuiGrid-grid-md-12 MuiGrid-grid-lg-12 MuiGrid-grid-xl-12"
        >
          <div
            className="MuiPaper-root MuiPaper-elevation1 MuiCard-root MuiPaper-rounded"
          >
            <div
              className="MuiCardHeader-root"
            >
              <div
                className="MuiCardHeader-content"
              >
                <span
                  className="MuiTypography-root MuiCardHeader-title MuiTypography-h5 MuiTypography-displayBlock"
                >
                  <div
                    className="MuiSkeleton-root MuiSkeleton-text MuiSkeleton-animate"
                    style={
                      Object {
                        "height": undefined,
                        "width": "100%",
                      }
                    }
                  />
                </span>
              </div>
            </div>
            <hr
              className="MuiDivider-root"
            />
            <div
              className="MuiCardContent-root"
            >
              <ul
                className="MuiList-root MuiList-padding"
              >
                <li
                  className="MuiListItem-root MuiListItem-gutters"
                  disabled={false}
                >
                  <div
                    className="MuiSkeleton-root MuiSkeleton-text MuiSkeleton-animate"
                    style={
                      Object {
                        "height": undefined,
                        "width": "100%",
                      }
                    }
                  />
                </li>
                <li
                  className="MuiListItem-root MuiListItem-gutters"
                  disabled={false}
                >
                  <div
                    className="MuiSkeleton-root MuiSkeleton-text MuiSkeleton-animate"
                    style={
                      Object {
                        "height": undefined,
                        "width": "100%",
                      }
                    }
                  />
                </li>
                <li
                  className="MuiListItem-root MuiListItem-gutters"
                  disabled={false}
                >
                  <div
                    className="MuiSkeleton-root MuiSkeleton-text MuiSkeleton-animate"
                    style={
                      Object {
                        "height": undefined,
                        "width": "100%",
                      }
                    }
                  />
                </li>
              </ul>
            </div>
          </div>
        </div>
        <div
          className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-12 MuiGrid-grid-md-12 MuiGrid-grid-lg-12 MuiGrid-grid-xl-12"
        >
          <div
            className="MuiPaper-root MuiPaper-elevation1 MuiCard-root MuiPaper-rounded"
          >
            <div
              className="MuiCardHeader-root"
            >
              <div
                className="MuiCardHeader-content"
              >
                <span
                  className="MuiTypography-root MuiCardHeader-title MuiTypography-h5 MuiTypography-displayBlock"
                >
                  <div
                    className="MuiSkeleton-root MuiSkeleton-text MuiSkeleton-animate"
                    style={
                      Object {
                        "height": undefined,
                        "width": "100%",
                      }
                    }
                  />
                </span>
              </div>
            </div>
            <hr
              className="MuiDivider-root"
            />
            <div
              className="MuiCardContent-root"
            >
              <ul
                className="MuiList-root MuiList-padding"
              >
                <li
                  className="MuiListItem-root MuiListItem-gutters"
                  disabled={false}
                >
                  <div
                    className="MuiSkeleton-root MuiSkeleton-text MuiSkeleton-animate"
                    style={
                      Object {
                        "height": undefined,
                        "width": "100%",
                      }
                    }
                  />
                </li>
                <li
                  className="MuiListItem-root MuiListItem-gutters"
                  disabled={false}
                >
                  <div
                    className="MuiSkeleton-root MuiSkeleton-text MuiSkeleton-animate"
                    style={
                      Object {
                        "height": undefined,
                        "width": "100%",
                      }
                    }
                  />
                </li>
                <li
                  className="MuiListItem-root MuiListItem-gutters"
                  disabled={false}
                >
                  <div
                    className="MuiSkeleton-root MuiSkeleton-text MuiSkeleton-animate"
                    style={
                      Object {
                        "height": undefined,
                        "width": "100%",
                      }
                    }
                  />
                </li>
              </ul>
            </div>
          </div>
        </div>
      </div>
    </div>
  </main>
</div>
`;
